src/nvim/log.h

```diff
@@ -69,6 +69,32 @@
 
 #endif
 
+#if __linux__ && MIN_LOG_LEVEL <= DEBUG_LOG_LEVEL
+#  include <execinfo.h>
+#  define DLOG_CALL_STACK(prefix) \
```

Comment:
I would think this wouldn't be used except for temporary debugging calls, right? So it could be `MIN_LOG_LEVEL <= INFO_LOG_LEVEL`, and named `LOG_CALLSTACK`.